buildout.cfg

```diff
@@ -15,9 +15,9 @@ eggs =
 defaults = ['-c', '--module', 'five.formlib']
 
 [sources]
-Zope2 = git https://github.com/zopefoundation/Zope
+Zope = git https://github.com/zopefoundation/Zope
 
 [versions]
-Zope2 =
+Zope =
```

Comment:
This should be done by `mr.developer` if it is actually needed.

Reply:
Deleted.